chainforge/react-server/src/backend/models.ts

```diff
@@ -1,5 +1,5 @@
-/** 
- * A list of all model APIs natively supported by ChainForge. 
+/**
+ * A list of all model APIs natively supported by ChainForge.
  */
 export type LLM = string | NativeLLM;
```

Comment:
I tried to split models from providers (i.e. have `type LLM = {provider : ..., model: string}`) but that seemed to be a relatively big change and I wasn't sure how that would impact saved flows. I'm curious to know what you think, I think Ollama and Huggingface would both benefit from this change.

Reply:
You're right that this should be refactored and the way I initially structured it was shortsighted given the combinatorial explosion of models we've seen since April. But refactoring might also mean impact on saved flows. `base_model` is internally equivalent to "provider," and I probably should have named that provider instead. 

Maybe this point should be added to the refactoring issue https://github.com/ianarawjo/ChainForge/issues/179